tests/spec/core/pluralize-spec.js

```diff
@@ -18,7 +18,6 @@ describe("Core - Pluralize", () => {
 
     const dfnFoo = doc.querySelectorAll("#section dfn")[0];
     expect(dfnFoo.id).toBe("dfn-foo");
-    expect(dfnFoo.dataset.lt).toBeFalsy();
```

Comment:
Do we need to make sure `data-lt` exists?

Reply:
Not as part of this module. It's set by `dfn.js` module IIRC.